config/database.yml

```diff
@@ -23,3 +23,7 @@ test:
 production:
   <<: *default
   database: db/production.sqlite3
+
+staging:
+  <<: *default
+  database: db/staging.sqlite3
```

Comment:
by default rails would use whatever is defined as `ENV['DATABASE_URL']` otherwise it would revert to whatever is defined in the database.yml. Since we're planning for the scaling now and migrating to AWS, I guess we need to edit the config, but `ENV['DATABASE_URL']` is always defined on Heroku.
http://edgeguides.rubyonrails.org/configuring.html#configuring-a-database